app/javascript/src/LoginPage/utils/formValidation.js

```diff
@@ -0,0 +1,65 @@
+// @flow
+
+import validate from 'validate.js'
+
+const constraintsTypes = {
+  text: {
+    presence: true,
+    length: {minimum: 1}
+  },
+  email: {
+    presence: true,
+    email: true,
+    length: {minimum: 1}
+  },
+  password: {
+    presence: true,
+    length: {minimum: 6}
+  },
+  password_confirmation: {
+    presence: true,
+    length: {minimum: 1},
+    equality: {
+      attribute: 'user[password]'
+    }
+  }
+}
+
+const constraints = {
```

Comment:
I'd rather have separate objects in each form, create a `validate` instance and inject it in the form props. Then, when the time of validation comes, you do it there and not here with a lot of data that is not obvious which one is important.

Another note, we should get the validation rules from Rails in order to avoid duplication. This can be achieved in a different PR.